locales/ca/generate.json

```diff
@@ -0,0 +1,52 @@
+{
+   "meta": {
+      "description": "Genera llistes amb les cançons que les teves bandes estan girant actualment",
+      "keywords": "Spotify, Setlist.fm, Generador de llistes, Festivals, Música, Concerts",
+      "title": "Generar | Festwrap"
+   },
+   "steps": {
+      "navigation": {
+         "finish": "Finalitzar",
+         "generate": "Generar",
+         "next": "Següent",
+         "previous": "Anterior"
+      },
+      "step1": {
+         "description": "Crea una llista nova o utilitza una llista propia existent",
+         "form": {
+            "createNewPlaylist": {
+               "description": "Utilitza aquesta opció per crear una nova llista de reproducció a Spotify",
```

Comment:
I believe that descriptions are quite redundant for both options: they do not provide additional information but just clutter the space. Should we remove them?